python/hail/methods/qc.py

```diff
@@ -234,22 +234,22 @@ def concordance(left, right):
     Table 1: Concordance statistics per column key
 
         - **s** -- column key.
-        - **nDiscordant** (*Long*) -- Count of discordant calls (see below for
+        - **n_discordant** (:py:data:`.tint64`) -- Count of discordant calls (see below for
           full definition).
-        - **concordance** (*Array[Array[Long]]*) -- Array of concordance per
```

Comment:
I wonder if it would make sense to use the new Hail type grammar in the docs. It would make more complex types like this easier to read, but you'd lose the hyperlinking.

Reply:
that's very reasonable.

Reply:
I suppose if we wanted the linking, we could just make it link to the root type, like
```
:class:`array[array[int64]] <.tarray>`
```

Reply:
Is it OK to leave this shift for a future day? Even if it looks verbose, at least it should be consistent now.

Reply:
Definitely. That's why I made it just a comment.

Reply:
OK, I think I addressed everything otherwise. Ready for another look.